DigitalLearningSolutions.Data.Tests/Services/TrackerServiceTests.cs

```diff
@@ -107,17 +107,29 @@ public void ProcessQuery_with_valid_action_correctly_serialises_null_response()
         }
 
         [Test]
-        public void ProcessQuery_with_GetObjectiveArrayCc_action_passes_query_params()
+        [TestCase("true", true)]
+        [TestCase("True", true)]
+        [TestCase("false", false)]

```

Comment:
Bah, missed this. We don't have a "False" case, so we may as well add that in. You can merge without rereview after that though.